objects/src/transaction/proven_tx.rs

```diff
@@ -288,6 +277,9 @@ impl ProvenTransactionBuilder {
 /// Describes the changes made to the account state resulting from a transaction execution.
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct TxAccountUpdate {
+    /// A unique identifier for the transaction, see [TransactionId] for additional details.
+    transaction_id: TransactionId,
```

Comment:
Why did we move `transaction_id` into `TxAccountUpdate`? I think of it more as of a property of the transaction rather than account update.

Reply:
We need to provide transaction ID in several different places where we have only `TxAccountUpdate`. I have looked at the node's code again and now I'm thinking, I can return it back and get it different way, thank you!